### PR TITLE
Encryption of .env file to keey it sync

### PR DESCRIPTION
**Pull request by etyloppihacilem**
This introduce a new way of securely syncing .env file without revealing secrets. Just put the `.passwd` file at the root of the repository, run
```bash
make install_hooks
```
to install pre-commit and post-merge hooks, to keep the encrypted env_file up to date and synced.

This introduces other new makefile rules, as `save_env` and `load_env` to encrypt or decrypt .env file.

### CODE REVIEW

File: scripts/pre-commit.sh
Comment (nerve-icaloss) [inline scripts/pre-commit.sh:1]: What is the point of the .NOTenv_file

**Review by nerve-icaloss — CHANGES_REQUESTED** [review]
Missing use case to when rebasing Forward branch to feature branch

**Comment by etyloppihacilem** [comment]
Imo the rebase is not needed as the post-merge hook would be called at every stage of a rebase, and then the pre-commit hook will make you save it...

**Comment by nerve-icaloss** [comment]
lgtm :1st_place_medal:

**Review by nerve-icaloss — APPROVED** [review]
lgtm

**Pull request merged**